its/ruling/src/test/java/org/sonar/java/it/JavaRulingTest.java

```diff
@@ -38,15 +39,10 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
We don't use import '*', in the https://github.com/SonarSource/sonar-developer-toolset there's a configuration saying:
```
Class count to use import with '*'" -> 999
```